train.py

```diff
@@ -0,0 +1,117 @@
+import os
+import time
+
+import hydra
+import torch
+import psutil
+import imageio
+import numpy as np
+from omegaconf import DictConfig
+from torchvision.utils import save_image
+
+import wandb
+from utils import get_interpolations
+from models.autoencoder import Autoencoder
+from train_utils import test_epoch, train_epoch
+
+
+def initialize_model(args: DictConfig):
+    if args.model.type == "AE":
+        model = Autoencoder(args)
+    else:
+        raise NotImplementedError("Model type not implemented")
+    return model
```

Comment:
```suggestion
def initialize_model(cfg: DictConfig):
    if cfg.model.type == "AE":
        model = Autoencoder(cfg)
    else:
        raise NotImplementedError("Model type not implemented")
    return model
```

Let's try to defaults to calling this object `cfg` everywhere - that's what we're using in other projects and it's great to unify names like that :)